python/core/auto_generated/vectortile/qgsvectortilelayer.sip.in

```diff
@@ -123,6 +123,15 @@ Sets renderer for the map layer.
     QgsVectorTileRenderer *renderer() const;
 %Docstring
 Returns currently assigned renderer
+%End
+
+    void setTileBorderRenderingEnabled( bool enabled );
```

Comment:
Wouldn't this be better as a render context flag?

Reply:
So it would be a global flag applied to all layers? And probably to raster tiles as well?
I don't know... I would probably prefer to keep it as is, maybe it can be moved to render context later. There are few more things like having tile XYZ numbers printed or border line/color configuration that would be nice to have too, but not sure if they all belong to render context.

Reply:
Yeah, it just struck me that it applies to other layers/providers too so would be nice to have something less ad-hoc to handle it. Could we hide from sip at least so it's not stable API?

Reply:
Hmm... removing it from sip would kind of remove the whole point of having it in API at all - I wanted to be able to turn this flag on/off from python console. I would be also fine with removal of this flag before the release if you don't like it, but now it is helping me with development... :-)

As a side note - it would be probably reasonable to mark vector tile layer related stuff as experimental API for now and pronounce it stable e.g. for 3.16 ...?